blur_no-alt.php

```diff
@@ -1,51 +1,83 @@
-* Requires at least: 5.0
-* Author: Kerri Hicks
-* Author URI: https://kerri.is/
-* License: GPL v3 or later
-* License URI: https://www.gnu.org/licenses/gpl-3.0.html
-* Description: Blur images in the WordPress editor interface if they don't have alt text. Socially engineer your WordPress content developers to write alt text for images.
-* Text Domain: blur-no-alt
-* Update URI: false
-*/
-
-function blur_admin_theme_style() {
-    wp_enqueue_style('blur-admin-theme', plugins_url('blur_no-alt.css', __FILE__)) ;
-}
-add_action('admin_enqueue_scripts', 'blur_admin_theme_style') ;
-
+ * Blur No-Alt
+ *
+ * @package     blur_no-alt
+ * @author      Kerri Hicks
+ * @license     GPL-3.0+
+ *
+ * Plugin Name: Blur No-Alt
+ * Plugin URI: https://github.com/kerri-hicks/blur_no-alt
+ * Version: 0.93
+ * Requires at least: 5.0
+ * Author: Kerri Hicks
+ * Author URI: https://kerri.is/
+ * License: GPL v3 or later
+ * License URI: https://www.gnu.org/licenses/gpl-3.0.html
+ * Description: Blur images in the WordPress editor interface if they don't have alt text. Optionally blur images on front-end for logged-in users. Socially engineer your WordPress content developers to write alt text for images.
+ * Text Domain: blur-no-alt
+ * Update URI: false
+ */
 
+/**
+ * Plugin class
+ */
 class BlurNoAltMessageDisplay {
+	/**
+	 * Plugin Version
+	 *
+	 * @var string
+	 */
+	private $plugin_version = '0.93';
+
+	/**
+	 * Used to store plugin options?
+	 *
+	 * @var array
+	 */
 	private $blur_no_alt_message_display_options;
+
+	/**
+	 * Default options for the plugin when not or partially defined
+	 *
+	 * @var array
+	 */
 	private $blur_no_alt_message_display_options_default = array(
-		'show_or_hide_0' => 'Hide'
+		'show_or_hide_0' => 0,
+		'blur_on_front'  => 0,
 	);
 
+	/**
+	 * Register all actions class is instantiated
+	 */
 	public function __construct() {
 		add_action( 'admin_menu', array( $this, 'blur_no_alt_message_display_add_plugin_page' ) );
 		add_action( 'admin_init', array( $this, 'blur_no_alt_message_display_page_init' ) );
-		add_action('admin_enqueue_scripts', array( $this, 'blur_admin_theme_style_blur_message') );
+		add_action( 'admin_enqueue_scripts', array( $this, 'blur_admin_styles' ) );
+		add_action( 'wp_enqueue_scripts', array( $this, 'blur_front_end_styles' ) );
 	}
 
+	/**
+	 * Register settings page
+	 */
 	public function blur_no_alt_message_display_add_plugin_page() {
 		add_options_page(
-			__( 'Blur No-Alt Message Display', 'blur-no-alt' ), // page_title
-			__( 'Blur No-Alt Message Display', 'blur-no-alt' ), // menu_title
-			'manage_options', // capability
-			'blur-no-alt-message-display', // menu_slug
-			array( $this, 'blur_no_alt_message_display_create_admin_page' ) // function
+			esc_html__( 'Blur No-Alt Settings', 'blur-no-alt' ),
+			esc_html__( 'Blur No-Alt', 'blur-no-alt' ),
+			'manage_options',
+			'blur-no-alt-message-display',
+			array( $this, 'blur_no_alt_message_display_create_admin_page' ),
```

Comment:
Getting rid of "Message Display" makes lot of sense here. I'd like to keep the comments, though. OK, maybe //function isn't necessary. :-) But a reminder in comments of what these components do is useful to anyone who might want to fork and make changes, or you know, me forgetting what they do a year later.

Reply:
This is interesting feedback. I found these comments distracting as they were often creating wrapped lines for me. I also had the documentation open and my VS Code shows the parameters when hovering over the functions:

![image](https://github.com/kerri-hicks/blur_no-alt/assets/1238696/6b54bf4d-b44a-4a40-aeff-780dedad9316)

So if you want these added back, I think I'd actually prefer it if they were _more_ verbose to really explain what was going on. Something like this:

```php
add_options_page(
	// the settings page title
	esc_html__( 'Blur No-Alt Settings', 'blur-no-alt' ),
	// the menu item in the Settings submenu
	esc_html__( 'Blur No-Alt', 'blur-no-alt' ),
	// capability required to view Settings page
	'manage_options',
	// URL slug for the settings page
	'blur-no-alt-message-display',
	// callback function to generate page output
	array( $this, 'blur_no_alt_message_display_create_admin_page' ),
);
```

What about that?

Reply:
I've [added comments back in the style I proposed](https://github.com/kerri-hicks/blur_no-alt/pull/9/files#diff-0bca8f9d8daff35959e717f9cf5541d3bb22f5a24e026f1e1cd2cdda8ec502a4R65). If that works for you, we can resolve this issue :)